components/org.wso2.carbon.status.dashboard.web/src/gadgets/chart-components/PieChart.jsx

```diff
@@ -165,12 +148,10 @@ export default class PieCharts extends React.Component {
             orientation,
             legend,
             scatterPlotRange,
-            randomUpdater
+            randomUpdater,
```

Comment:
comma is not needed. please check

Reply:
Fixed in bbb066d61f625e5e28b09b91ec2e5a2dc09ed919 .Move to separate repo.